diff -r -u source-orig/src/libstore/derivations.cc source/src/libstore/derivations.cc
--- source-orig/src/libstore/derivations.cc	2018-03-15 15:44:58.428220077 -0300
+++ source/src/libstore/derivations.cc	2018-03-15 16:08:29.962811240 -0300
@@ -5,6 +5,8 @@
 #include "worker-protocol.hh"
 #include "fs-accessor.hh"
 #include "istringstream_nocopy.hh"
+#include <list>
+#include <algorithm>
 
 namespace nix {
 
@@ -56,6 +58,8 @@
 
 bool BasicDerivation::canBuildLocally() const
 {
+    auto platforms = settings.buildExtraPlatforms.get();
+    if (std::find(platforms.begin(), platforms.end(), platform) != platforms.end()) return true;
     return platform == settings.thisSystem
         || isBuiltin()
 #if __linux__
diff -r -u source-orig/src/libstore/globals.hh source/src/libstore/globals.hh
--- source-orig/src/libstore/globals.hh	2018-03-15 15:44:58.428220077 -0300
+++ source/src/libstore/globals.hh	2018-03-15 15:54:19.801648299 -0300
@@ -373,6 +373,9 @@
 
     Setting<Paths> pluginFiles{this, {}, "plugin-files",
         "Plugins to dynamically load at nix initialization time."};
+
+    Setting<Strings> buildExtraPlatforms{this, {}, "build-extra-platforms",
+      "A list of system platforms nix will run binaries for"};
 };
 
 
